tests/baselines/reference/parserRealSource8.errors.txt

```diff
@@ -397,7 +397,7 @@ tests/cases/conformance/parser/ecmascript5/parserRealSource8.ts(454,35): error T
 !!! error TS2304: Cannot find name 'Type'.
             var withSymbol = new WithSymbol(withStmt.minChar, context.typeFlow.checker.locationInfo.unitIndex, withType);
                                  ~~~~~~~~~~
-!!! error TS2304: Cannot find name 'WithSymbol'.
+!!! error TS2552: Cannot find name 'WithSymbol'. Did you mean 'withSymbol'?

```

Comment:
I'd only give this one an A because running the code fix would produce `var withSymbol = new withSymbol(....)`. And this differ-only-by-case pattern is common, so maybe only a B+.